test/FunctionalTests/Tests/DataServices/UnitTests/ClientCSharpUnitTests/DataWebClientCSharp/LinqTests.cs

```diff
@@ -8153,6 +8153,34 @@ public void QueryRowCount_UriBuilder()
             }
         }
 
+        [TestMethod]
+        public void VerifyCountIsNotCarriedOver()
+        {
+            string expectedLongCountFunctionUri = "/service.svc/Leagues/LongCountFunction(Param1='Param1',Param2='Param2')?$orderby=ID";
+            string expectedCountFunctionUri = "/service.svc/Leagues/CountFunction(Param1='Param1',Param2='Param2')?$orderby=ID";
+
+            var countQuery = context.CreateQuery<League>("Leagues");
+            var longcountQuery = context.CreateQuery<League>("Leagues");
+
+            var countFunctionQuery = countQuery.CreateFunctionQuery<League>("CountFunction", true, new UriOperationParameter("Param1", "Param1"),
+                    new UriOperationParameter("Param2", "Param2"));
+            var longCountFunctionQuery = longcountQuery.CreateFunctionQuery<League>("LongCountFunction", true, new UriOperationParameter("Param1", "Param1"),
+                    new UriOperationParameter("Param2", "Param2"));

```

Comment:
I personally prefer vars over types but the style in this file is to use the type.
Var could also be used if the whole method adhered to this but can see that we started by declaring a couple of ```string```s. Maybe those could be replaced and then the ```[TestMethod]VerifyCountIsNotCarriedOver()``` becomes internally consistent.

Reply:
Fixed.